.github/workflows/build-ramdisk.yml

```
@@ -3,37 +3,43 @@ name: Build and Test with Ramdisk
+      - main
   pull_request:
 
 jobs:
   build-and-test:
     runs-on: ubuntu-latest
 
     steps:
+      # Step 1: Checkout the repository
       - name: Checkout Code
-        uses: actions/checkout@v6
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)